build/images/scripts/restart_sandbox

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+CRICTL_COMMAND="crictl --runtime-endpoint unix:///run/containerd/containerd.sock"
+
+if [ -f /etc/cni/net.d/10-antrea.conflist ]; then
+    mv /etc/cni/net.d/10-antrea.conflist /etc/cni/net.d/00-antrea.conflist
+fi
+# Remove rules of other CNI in CHAIN CNI-HOSTPORT-DNAT.
+# CHAIN CNI-HOSTPORT-DNAT is created by portmap for ingress controllers. It will not be deleted
+# even if the CNI is removed, so we delete the stale rules manually.
+if iptables -t nat -S | grep -q "CNI-HOSTPORT-DNAT"; then
+    rules=$(iptables -t nat -L CNI-HOSTPORT-DNAT --line-numbers | grep -v "antrea" | awk 'NR>2 {print $1}')
+    for rule in $rules; do
+        iptables -t nat -D CNI-HOSTPORT-DNAT "$rule"
+    done
+    chains=$(iptables -t nat -L CNI-HOSTPORT-DNAT | grep -v "antrea" | awk 'NR>2 {print $1}')
```

Comment:
What does the two list mean?
Doesn't the first step already remove all rules? how could the second list get anything?

Reply:
Yes, this is incorrect. This line should be moved right below the line 28, then  CHAINS for other CNIs will be removed. Fixed.